ui1.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="author" content="Ozan Paker">
+    <meta name="keywords" content="html pratikleri,html">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML-Practices</title>
+    <!-- title ne işe yarar? -->
+    <link id="favicon" rel="shortcut icon" type="image/png" href="favicon.png"/>
+  </head>
+  <body>
+    <h1>Html Öğrenme</h1>
+    <h2>Html Öğrenme</h2>
+    <h3>Html Öğrenme</h3>
+    <h4>Html Öğrenme</h4>
+    <h5>Html Öğrenme</h5>
+    <h6>Html Öğrenme</h6>
+    <hr/>
+    <br/>
+    <a href="https://www.google.com.tr" target="_blank">hebele hübele</a>
```

Comment:
a Yani anchor için 2 örnek yapmışız ama _blank yerine kullanılabilecekleri görmüyoruz buraya belki diğer 3 ünü de eklemek güzel olabilirdi 🤔